napari/benchmarks/benchmark_vectors_layer.py

```diff
@@ -92,7 +92,7 @@ def time_length(self, n):
         """Time to update length."""
         self.layer.length = 2
 
-    def mem_layer(self, n):
+    def _mem_layer(self, n):
```

Comment:
What is the effect of this? Does this mean the memory benchmarks won't be run?

Reply:
ah, just saw the updated description...

Reply:
Maybe add todo comments on each changed line? It will be weird to come across them otherwise

Reply:
My plan was to.open followup PR really fast. Bu I will add comment.